easybuild/framework/easyblock.py

```diff
@@ -1640,6 +1658,7 @@ def patch_step(self, beginpath=None):
         """
         for patch in self.patches:
             self.log.info("Applying patch %s" % patch['name'])
+            trace_msg("applying patch %s..." % patch['name'])
```

Comment:
but here I would add a space after the dots.

Reply:
Why, each call to `trace_msg` prints a separate line, what's the use of having a space after `...` if it's followed by a `\n`?

Reply:
I mean a seperate between the filename of the patch and the dots. Or why the dots at all?

Reply:
OK, so before the dots. ;-)

But yeah, just dropping the dots makes sense, will change.